src/modules/Builder/features/BuilderAppletSettings/LeftBar/Item/Item.styles.ts

```diff
@@ -0,0 +1,66 @@
+import { styled } from '@mui/system';
+
+import {
+  theme,
+  variables,
+  StyledFlexColumn,
+  StyledFlexTopCenter,
+  StyledTitleBoldMedium,
+  StyledTitleMedium,
+} from 'shared/styles';
+import { shouldForwardProp } from 'shared/utils';
+
+const commonEllipsisStyles = `
```

Comment:
please add these common styles into `src/shared/styles/stylesConsts.ts` , they are also used in StyledEllipsisText